bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -11,6 +11,9 @@ parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument('--signal-file')
 parser.add_argument('--template-file', required=True)
 parser.add_argument('--min-mass', type=float, default=None, required=True)
+parser.add_argument('--fit-param', nargs='+', required=True,
+                    help='Parameters over which KDE is calculated')
```

Comment:
As discussed, the input param names should be expected to be just commonly used strings like eg 'mchirp', 'eta', 'q', 'chi_eff', 'mtotal' etc, which can be evaluated by feeding the template bank and the parameter name to the bank conversions function or by feeding the signal masses and spins and the parameter name to the triggers.get_param function.